packages/cli/src/databases/mysqldb/migrations/1636626154933-CreateUserManagement.ts

```diff
@@ -0,0 +1,158 @@
+			'`userId` VARCHAR(100) NOT NULL, ' +
+			'`credentialsId` INT NOT NULL, ' +
+			'INDEX `FK_c68e056637562000b68f480815a_idx` (`roleId` ASC) VISIBLE, ' +
+			'INDEX `FK_484f0327e778648dd04f1d70493_idx` (`userId` ASC) VISIBLE, ' +
+			'PRIMARY KEY (`userId`, `credentialsId`), ' +
+			'CONSTRAINT `FK_c68e056637562000b68f480815a` ' +
+			  'FOREIGN KEY (`roleId`) ' +
+			  'REFERENCES `' + tablePrefix + 'role` (`id`) ' +
+			  'ON DELETE NO ACTION ' +
+			  'ON UPDATE NO ACTION, ' +
+			'CONSTRAINT `FK_484f0327e778648dd04f1d70493` ' +
+			  'FOREIGN KEY (`userId`) ' +
+			  'REFERENCES `' + tablePrefix + 'user` (`id`) ' +
+			  'ON DELETE CASCADE ' +
+			  'ON UPDATE NO ACTION, ' +
+			'CONSTRAINT `FK_68661def1d4bcf2451ac8dbd949` ' +
+			  'FOREIGN KEY (`credentialsId`) ' +
+			  'REFERENCES `' + tablePrefix + 'credentials_entity` (`id`) ' +
+			  'ON DELETE NO ACTION ' +
+			  'ON UPDATE NO ACTION);'
+		);
+
+		await queryRunner.query(
+			'CREATE TABLE `' + tablePrefix + 'settings` ( ' +
+				'`key` VARCHAR(250) NOT NULL, ' +
+				'`value` TEXT(10000) NOT NULL, ' +
+				'`loadOnStartup` TINYINT(1) NOT NULL DEFAULT 0, ' +
+				'PRIMARY KEY (`key`));'
+		);
+
+		// Insert initial roles
+		await queryRunner.query('INSERT INTO `' + tablePrefix + 'role` (name, scope) VALUES ("owner", "global");');
+
+		const instanceOwnerRole = await queryRunner.query('SELECT LAST_INSERT_ID() as insertId');
+
+		await queryRunner.query('INSERT INTO `' + tablePrefix + 'role` (name, scope) VALUES ("member", "global");');
+
+		await queryRunner.query('INSERT INTO `' + tablePrefix + 'role` (name, scope) VALUES ("owner", "workflow");');
+
+		const workflowOwnerRole = await queryRunner.query('SELECT LAST_INSERT_ID() as insertId');
+
+		await queryRunner.query('INSERT INTO `' + tablePrefix + 'role` (name, scope) VALUES ("owner", "credential");');
+
+		const credentialOwnerRole = await queryRunner.query('SELECT LAST_INSERT_ID() as insertId');
+
+		const survey = loadSurveyFromDisk();
+
+		const ownerUserId = uuid();
+		await queryRunner.query(
+			'INSERT INTO `' + tablePrefix + 'user` ' +
+			  '(id, globalRoleId, personalizationAnswers) values  ' +
+				'(?, ?, ?)',
+				[ownerUserId, instanceOwnerRole[0].insertId, survey ?? null]
+		);
+
+		await queryRunner.query(
+			'INSERT INTO `' + tablePrefix + 'shared_workflow` (createdAt, updatedAt, roleId, userId, workflowId)  ' +
+			' select NOW(), NOW(), "' + workflowOwnerRole[0].insertId + '", "' + ownerUserId + '", id from `' + tablePrefix + 'workflow_entity`'
+		);
+
+		await queryRunner.query(
+			'INSERT INTO `' + tablePrefix + 'shared_credentials` (createdAt, updatedAt, roleId, userId, credentialsId)  ' +
+			' select NOW(), NOW(), "' + credentialOwnerRole[0].insertId + '", "' + ownerUserId + '", id from `' + tablePrefix + 'credentials_entity`'
+		);
+
+		await queryRunner.query(
+			'INSERT INTO `' + tablePrefix + 'settings` (`key`, value, loadOnStartup) values  ' +
+			'("userManagement.hasOwner", "false", 1)'
+		);
```

Comment:
we need to add to index the `workflowId` on executions:

`CREATE INDEX "IDX_xeendlvptc5jy4hbol17b5xery" ON "execution_entity" ("workflowId") `,